specification/compute/resource-manager/readme.go.md

```diff
@@ -2,7 +2,7 @@
 
 These settings apply only when `--go` is specified on the command line.
 
-```yaml $(go)
+```yaml $(go) && !$(track2)
```

Comment:
Astonishing that I have add this, otherwise the directive below will override this completely